### PR TITLE
Functionality for adding markers manually

### PR DESCRIPTION
**Pull request by AditiSaini**

### CODE REVIEW

File: src/main/webapp/user-page.html [hunk 1]
```diff
@@ -138,6 +141,9 @@ <h6 class="font-weight-lighter text-muted" id="aboutme"></h6>
             Loading...
           </div>
         </div>
+          <div class="text-center add-marker-manually">
```
Comment (amad-person) [inline src/main/webapp/user-page.html]: This should probably be moved above the messages area - if there are too many messages, the user can miss this

Comment (amad-person) [reply src/main/webapp/user-page.html]: @anqitu let us know what you think since you're great at styling! 😄

Comment (anqitu) [reply src/main/webapp/user-page.html]: Sorry, but may I ask what is this button supposed to do?

Comment (anqitu) [reply src/main/webapp/user-page.html]: I thought the location is specific for each image?

Comment (AditiSaini) [reply src/main/webapp/user-page.html]: Sometimes when the location of the image can't be detected, it allows the users to add manually

Comment (anqitu) [reply src/main/webapp/user-page.html]: Maybe we can use the Map button on the post message div or/and add a button for each post message? I feel it is a bit confusing if we put the button there. As it should be for a single message, and not for a user profile config.

**Comment by amad-person** [comment]
@AditiSaini the autocomplete works well, but the marker isn't showing up on the map. I added a location in the text box and clicked share, is this what I was supposed to do or do I need to do something else? 

Also when I add a marker from the Map page and refresh the page, the map doesn't show the marker I added.

File: src/main/webapp/user-page.html (hunk 1 above)
Comment (AditiSaini) [reply src/main/webapp/user-page.html]: Made the changes :)

**Review by anqitu — APPROVED** [review]
Cool!

**Pull request merged**